.jenkins/jenkins_pod.yml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Pod
+spec:
+  imagePullSecrets:
+    - name: talend-registry
+  containers:
+    - name: main
+      # 3.0.9-20220930152032 is the last root version of custom-builder
+      image: 'artifactory.datapwn.com/tlnd-docker-dev/talend/common/tsbi/custom-builder:3.0.9-20220930152032'
```

Comment:
Why custom-builder release is not following latest tbsi release?
AFAIK, it was still using layerspector, no?

Reply:
There is a big job to bump above 3.0.9-20220930152032 because the Jenkins is not root any more.
So this is a first step to merge, then I will plan a new ticket to go further.

Reply:
As explained in the doc https://github.com/Talend/tsbi-images/tree/main/tsbi/custom-builder, This image is intended to be the only image used on CI and should contain everything needed to build our products and run our automation scripts.